packages/donor/src/screens/questionnaire/questions/QuestionsIchilov.tsx

```diff
@@ -102,7 +102,9 @@ export default function QuestionsIchilov({
         }
         buttonApproveText="שלח/י ואטסאפ לרכז שלך"
         onApproved={() => {
-          window.open(WHATSAPP_LINK);
+          window.open(
+            `${WHATSAPP_LINK}?text=אהלן, לפי השאלון באפליקציה אין לי יכולת לתרום טרומבוציטים כרגע, כיצד אוכל לשנות זאת?`
```

Comment:
I suggest we create a util function that accepts a message and creates the WA link (instead of constructing the string in many places).

What do you think?

If you agree, I would put it in common cause it may be useful in the functions in the coming future

Reply:
I agree, take a look at the new changes.